eventlet/greenthread.py

```diff
@@ -229,11 +231,12 @@ def _resolve_links(self):
         # ca and ckw are the curried function arguments
         if self._resolving_links:
             return
+        if not self._exit_funcs:
+            return
```

Comment:
If `_exit_funcs` is always a `deque`, this early exit becomes optional, and its value depends on anticipated usage. But I could live with the assumption that more `GreenThread`s are run without `link()` calls than with `link()` calls.